docs/testbed/README.testbed.OCS.md

```diff
@@ -0,0 +1,69 @@
+# OCS Topology Definition
+
+## Table of Contents
+  * [Revision History](#revision-history)
+  * [Objective](#objective)
+  * [Physical Topology](#physical-topology)
+  * [Logical Topology](#logical-topology)
+  * [Testbed Requirements](#testbed-requirements)
+
+## Revision History
+
+| Date       | Author    | Description   |
+| ---------- | --------- | ------------- |
+| 2025-07-09 | Huang Xin | Initial draft |
+
+## Objective
+This document defines the topology of the Optical Circuit Switch (OCS) to verify the basic functionality of the OCS device. 
+
+## Physical Topology
+![OCS Topology](docs/testbed/img/testbed-ocs_topology.png)
```

Comment:
The image cannot be rendered in preview of this markdown file. Can you double check and fix?